Replace multi-colored icons with components (#3757)
sumup-oss/circuit-ui: .changeset/ripe-rivers-smoke.md

---
"@sumup-oss/icons": minor
---

> Added multi-colored PaymentMethod and CardScheme components to icons package. They were previously available as SVG icon components, but inlining their full-color SVG source made bundles heavy, so these components load the icon as an image instead.

Reviewer: ```suggestion
Added `PaymentMethod` and `CardScheme` components. They were previously available as individual SVG icon components, but inlining their full-color SVG source made bundles heavy, so these components load the icons as an image from a URL instead.
```
